### DISCUSSION TITLE
Add custom metaData from Jenkins job

### DISCUSSION THREAD

**Discussion by peacetrader** [WasiqB/multiple-cucumber-html-reporter#330] [Q&A] [answered]
Hi, currently we can add metaData into html report. In the below block, the release value or release build has to be called from another Jenkins job which has build number and that build number is the build tag in TEST environment. I would like to know if we can incorporate this into the html report rather than hardcoding it. 
customData: {
    title: 'Run info',
    data: [
        {label: 'Project', value: 'Custom project'},
        {label: 'Release', value: '1.2.3'},
        {label: 'Execution Start Time', value: 'Nov 19th 2017, 02:31 PM EST'},
        {label: 'Execution End Time', value: 'Nov 19th 2017, 02:56 PM EST'}
    ]
}

**Comment by WasiqB** [comment, marked as answer]
Hi @peacetrader ,

New version [4.0.0-beta.0](https://github.com/WasiqB/multiple-cucumber-html-reporter/releases/tag/v4.0.0-beta.0) has been released yesterday.

Check out the [sample implementation](https://github.com/WasiqB/multiple-cucumber-html-reporter/blob/d349070f440393ba9227d154953723264d4be56b/examples/wdio/wdio-shared.conf.ts#L343) where you can update the metadata dynamically.
